lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -444,7 +444,7 @@ namespace Js
                 inlineeOffset = inlinedFrameWalker.GetCurrentInlineeOffset();
             }
         }
-        else if (ScriptFunction::Is(parentFunction) && HasInlinedFramesOnStack())
+        else if (ScriptFunction::Test(parentFunction) && HasInlinedFramesOnStack())
```

Comment:
>ScriptFunction::Test( [](start = 17, length = 21)

there are couple of other places in `JavascriptObject.cpp`, `FixedFieldInfo.cpp` and `Bailout.cpp` where we do `ScriptFunction::Is`. Can you change those as well?